scripts/process_biomappings.py

```diff
@@ -0,0 +1,19 @@
+import pandas as pd
+# import sys
+
+# biomappings_file = sys.argv[1]
+biomappings_file = "https://raw.githubusercontent.com/biopragmatics/biomappings/master/docs/_data/sssom/biomappings.sssom.tsv"
+df = pd.read_csv(biomappings_file, sep='\t')
+
+# Get only chebi to mesh rows
+df = df[(df['subject_id'].str.contains('chebi')) & (df['object_id'].str.contains('mesh'))]
+
+# Capitalize the mesh and chebi ids
+df['subject_id'] = df['subject_id'].str.upper()
+df['object_id'] = df['object_id'].str.upper()
+
+# Swap values of subject id and object id
+df['subject_id'], df['object_id'] = df['object_id'].values, df['subject_id'].values
```

Comment:
We should use 

```
sssom invert ....
```

For this, because there is much more to flipping a mapping set then flipping the IDs.

Trivially, but not sufficiently, the subject_label and object_label columns must be flipped too, but there is more. Look at 

https://github.com/monarch-initiative/monarch-mapping-commons/blob/c12886f1c0e99740f0dc23bf09f095fb3224af12/mappings/biomappings.sssom.tsv#L1252

Flipping this the way you do will cause bigger errors!

Reply:
Thanks Nico! I think I've resolved the other two suggestions with this commit, but I'm having trouble with sssom invert... it looks like an invert mapping dictionary may be a required argument? but I'm not totally sure what that should look like